vault-plugins/flant_iam/backend/tests/specs/helpers.go

```diff
@@ -136,16 +152,95 @@ func CreateRandomRole(roleAPI api.TestAPI) model.Role {
 func CreateRandomProject(projectAPI api.TestAPI, tenantID model.TenantUUID) model.Project {
 	createPayload := fixtures.RandomGroupCreatePayload()
 	createPayload["tenant_uuid"] = tenantID
-
 	params := api.Params{
 		"tenant": tenantID,
 	}
 	createData := projectAPI.Create(params, url.Values{}, createPayload)
-
 	rawProject := createData.Get("project")
 	data := []byte(rawProject.String())
 	var project model.Project
 	err := json.Unmarshal(data, &project) //nolint:errcheck
 	Expect(err).ToNot(HaveOccurred())
 	return project
 }
+
+func CreateRoleBinding(rolebindingAPI api.TestAPI, rb model.RoleBinding) model.RoleBinding {
+	params := api.Params{
+		"tenant": rb.TenantUUID,
+	}
+	bytes, _ := json.Marshal(rb)
+	var createPayload map[string]interface{}
+	json.Unmarshal(bytes, &createPayload) //nolint:errcheck
+	createData := rolebindingAPI.Create(params, url.Values{}, createPayload)
+	rawRoleBinding := createData.Get("role_binding")
+	data := []byte(rawRoleBinding.String())
+	var roleBinding model.RoleBinding
+	err := json.Unmarshal(data, &roleBinding) //nolint:errcheck
```

Comment:
`//nolint:errcheck` is redundant here. +3 times below.

Reply:
fixed